src/components/pre-search/manage-pre-search.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div v-if="!$x.query.searchBox">
+    <h1 v-if="!hasQueryPreviews" class="x-title1 max-desktop:x-title1-sm max-desktop:x-px-16">
+      {{ $t('popularSearches.title') }}
+    </h1>
+    <CustomQueryPreview
+      :queriesPreviewInfo="
+        hasQueryPreviews ? queriesPreviewInfo : getQueriesPreviewInfo($x.popularSearches)
+      "
+    />
+  </div>
+</template>
+
+<script lang="ts">
+  import { computed, defineComponent, inject } from 'vue';
+  import { QueryPreviewInfo } from '@empathyco/x-components/queries-preview';
+  import { Suggestion } from '@empathyco/x-types';
+  import CustomQueryPreview from './custom-query-preview.vue';
+
+  export default defineComponent({
+    components: {
+      CustomQueryPreview
+    },
+    props: {
+      maxPopularSearchesToRender: {
+        type: Number,
+        default: 4
+      }
+    },
+    setup(props) {
+      const injectedQueriesPreview = inject<{ value: QueryPreviewInfo[] }>('queriesPreviewInfo', {
+        value: []
+      });
+      const queriesPreviewInfo = computed<QueryPreviewInfo[]>(() => injectedQueriesPreview.value);
+
+      const hasQueryPreviews = computed<boolean>(() => queriesPreviewInfo.value.length !== 0);
+
+      const getQueriesPreviewInfo = (suggestion: readonly Suggestion[]): QueryPreviewInfo[] => {
+        const queryPreviewInfo: QueryPreviewInfo[] = suggestion.map(item => ({
+          query: item.query
+        }));
+        return queryPreviewInfo.slice(0, props.maxPopularSearchesToRender);
+      };
```

Comment:
It's difficult to understand what this method is doing. It's called `getQueriesPreviewInfo`, but you already have a computed property called `queriesPreviewInfo`. It's not clear why it's needed or what it does.

I suggest convert it to a computed property, return the injected queriesPreview and if there's none then use the `useStore` composable (define it outside of the computed) to take the popular searches and convert them to queryPreview Info. 

Also, we need to register the popular searches module if we're going to use them.